doc/usage.rst

```diff
@@ -224,6 +230,12 @@ syntactic sugar::
   >>> target[FakeConsoleDriver, 'console']
   FakeConsoleDriver(target=Target(name='main', …), name='console', …)
 
+After you are done with the target, call the cleanup function to ensure
+drivers are deactivated and processes are stopped:::
+
+  >>> target.clean()
```

Comment:
The function name is cleanup(). If we go the atexit approach, this would be optional but could catch some errors earlier.

Reply:
I'll reword this to conform to this use case.